quipucords/api/deployments_report/serializer.py

```diff
@@ -84,11 +84,11 @@ class SystemFingerprintSerializer(NotEmptySerializer):
     system_creation_date = DateField(required=False)
     system_last_checkin_date = DateField(required=False)
 
-    system_intent_role = CharField(required=False, max_length=128)
-    system_intent_addons = JSONField(required=False)
-    system_intent_sla = CharField(required=False, max_length=128)
-    system_intent_usage_type = CharField(required=False, max_length=128)
-    system_intent_usage_type = CharField(required=False, max_length=128)
+    system_role = CharField(required=False, max_length=128)
+    system_addons = JSONField(required=False)
+    system_service_level_agreement = CharField(required=False, max_length=128)
+    system_usage_type = CharField(required=False, max_length=128)
+    system_usage_type = CharField(required=False, max_length=128)
```

Comment:
why is `system_usage_type` in here twice?

Reply:
Fixed.